src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -284,8 +284,18 @@ private void requireDockerApiVersionLessThan(final String required, final String
     assumeTrue(msg, dockerApiVersionLessThan(required));
   }
 
+  private boolean dockerApiVersionNot(final String expected) {
+    return compareVersion(dockerApiVersion, expected) != 0;
+  }
+
+  private void requireDockerApiVersionNot(final String version, final String msg) {
+    assumeTrue(msg, dockerApiVersionNot(version));
+  }
+
   @Test
   public void testSearchImage() throws Exception {
+    requireDockerApiVersionNot("1.19", "Docker 1.7.x sends the wrong Content-Type header for "
```

Comment:
I think you would have to add something to the jersey client config that told it to use Jackson/ObjectMapper when the content-type is text/plain, which feels wrong and likely to cause other bugs. It might be possible to add a filter of some sort that modifies the response header for this one URI before Jersey tries to read it based on the content-type, but I am not sure about that either. It may be best to leave as-is unless there is a strong desire for someone to use this method with 1.7.